src/components/organisms/AdminVenueView/AdminVenueView.scss

```diff
@@ -0,0 +1,52 @@
+@import "scss/constants.scss";
+
+.venue-view {
+  display: flex;
+  justify-content: space-around;
+  padding: 20px 0;
+
+  @media (max-width: 1024px) {
+    align-items: center;
+  }
+
+  @media (max-width: 800px) {
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+  }
```

Comment:
> Do we already have breakpoints in `constants.scss`

No.

> Can we standardize our breakpoints in `constants.scss`

Of course. Any preferences? Maybe follow grid's `sm md lg`?
Ex: `$media-width-sm`

Reply:
I thought @mcflyok may have already has some stuff for this in the style guide, but can't find it where I expected to:

- https://github.com/sparkletown/sparkle-styling/blob/main/sparkle-styleguide/scss/global.scss#L1-L25

General concept/chats we were having was based on bootstrap/grids sort of patterns. eg. see:

- https://github.com/sparkletown/internal-sparkle-issues/issues/195#issuecomment-764747242

While they aren't currently pulled out into variables, the comments here suggest what size each breakpoint is for:

- https://github.com/sparkletown/sparkle-styling/blob/main/sparkle-styleguide/scss/venues.scss

```scss
//Small Screens
@media (max-width: 576px)

//Medium Screens
@media (max-width: 768px)

//Large Screens
@media (min-width: 992px)

//Extra Large Screens
@media (min-width: 1200px)
```

Which seems to align to the bootstrap breakpoints:

- https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints
  - Small devices (landscape phones, 576px and up): `@media (min-width: 576px) { ... }`
  - Medium devices (tablets, 768px and up): `@media (min-width: 768px) { ... }`
  - Large devices (desktops, 992px and up): `@media (min-width: 992px) { ... }`
  - Extra large devices (large desktops, 1200px and up): `@media (min-width: 1200px) { ... }`
  - https://getbootstrap.com/docs/4.0/layout/grid/#grid-options

It might even be possible for us to just reuse the mixins:

- https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints

```scss
// min-width (main ones we would use)
@include media-breakpoint-up(xs) { ... }
@include media-breakpoint-up(sm) { ... }
@include media-breakpoint-up(md) { ... }
@include media-breakpoint-up(lg) { ... }
@include media-breakpoint-up(xl) { ... }

// max-width (only if required)
@include media-breakpoint-down(xs) { ... }
@include media-breakpoint-down(sm) { ... }
@include media-breakpoint-down(md) { ... }
@include media-breakpoint-down(lg) { ... }
```

---

If we need to create variables for it (though my preference is possibly just to reuse bootstraps mixins at this stage), so long as it follows some kind of 'reasonably descriptive' format, that's probably fine.

To follow bootstrap's naming:

- `$media-breakpoint--sm` or `$media-breakpoint--sm-up` / `$media-breakpoint--sm-down` (note that the 'down' are using fractional sizes so they don't both apply)
- etc

Reply:
Somewhat OOS of this PR, but I would probably also change that `$small-laptop-width` to using the mixins/etc method above eventually too.

Reply:
I this this should work:

$media-breakpoint--sm = 576px
$media-breakpoint--md = 768px
$media-breakpoint--lg = 992px
$media-breakpoint--xl = 1200px

Reply:
Should we just import/re-export/use the bootstrap mixins mentioned in https://github.com/sparkletown/sparkle/pull/1190#discussion_r592012437

It sounds like they are designed to avoid some 'overlapping pixel' issues/etc.

Reply:
Oh, I missed that we already have `bootstrap` imported. Yep, reusing it seems like a better approach